ocfweb/tv/templatetags/tv_formatting.py

```diff
@@ -0,0 +1,14 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def tv_lab_hours(hour):
+    if hour:
+        if hour.open.minute != 0 or hour.close.minute != 0:
+            fmt = '{:%-I:%M%P}-{:%-I%M%P}'
+        else:
+            fmt = '{:%-I%P}-{:%-I%P}'
```

Comment:
probably want en-dashes like the other filter, instead of the hyphens you're using now?

8am-9pm
vs
8am–9pm